components/tag-list/tag-list-item-mixin.js

```diff
@@ -27,6 +27,11 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 			 * @type {boolean}
 			 */
 			clearable: { type: Boolean },
+			/**
+			 * Acts as a unique identifier for the tag
+			 * @type {string}
+			 */
+			key: { type: String },
```

Comment:
Do we want to make this as "Required" in the docs? 🤔 It doesn't really break anything if you don't have it, but the event _does_ expect it.